packages/mobile/src/fees/selectors.ts

```diff
@@ -0,0 +1,47 @@
+import BigNumber from 'bignumber.js'
+import { createSelector } from 'reselect'
+import { FeeType } from 'src/fees/actions'
+import { RootState } from 'src/redux/reducers'
+import { divideByWei } from 'src/utils/formatting'
+
+const getInviteFeeEstimate = (state: RootState) => state.fees.invite.feeInWei
```

Comment:
For clarity, can we append `InWei` to these names?

Reply:
Yes!